images/dog-list.html

```diff
@@ -0,0 +1,205 @@
+                        <h3>Violet</h3>
+                        <h4>Cost To Adopt:</h4>
+                        <p>$123.45</p>
+                        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. </p>
+                        <button>
+                            Adopt
+                        </button>
+                    </article>
+                    <article>
+                        <img src="maximus-card.jpg" alt="Maximus"> 
+                        <h3>Violet</h3>
+                        <h4>Cost To Adopt:</h4>
+                        <p>$123.45</p>
+                        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. </p>
+                        <button>
+                            Adopt
+                        </button>
+                    </article>
+                    <article>
+                        <img src="luna-card.jpg" alt="Luna"> 
+                        <h3>Violet</h3>
+                        <h4>Cost To Adopt:</h4>
+                        <p>$123.45</p>
+                        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. </p>
+                        <button>
+                            Adopt
+                        </button>
+                    </article>
+                     <article>
+                        <img src="stella-card.jpg" alt="Stella"> 
+                        <h3>Violet</h3>
+                        <h4>Cost To Adopt:</h4>
+                        <p>$123.45</p>
+                        <p> Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. </p>
+                        <button>
+                            Adopt
+                        </button>
+                    </article>
+                </div>
+            </section>
+            <button>
+                See More Dogs
+            </button>
+        </main>
+        <footer>
+            <section>
+                <!--Logo here-->
+                <img src="adoptadog-logo.svg" alt="Adoptadog">
+            </section>
+            <section>
+                <p>
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
+                </p>
+            </section>
+            <section>
+                 <ul>
+                    <li> <a href="images/index.html">Home</a> </li>
+                    <li> <p>/</p> </li>
+                    <li> <a href="images/dog-list.html">Dogs</a> </li>
+                    <li> <p>/</p> </li>
+                    <li> <a href="images/blog.html">Blog</a> </li>
+                </ul>
+            </section>
+            <address>
+                <p>Adoptadog, LLC | Seattle, WA</p>
```

Comment:
you can add the copyright symbol using `&copy;`

` <p>&copy; Adoptadog, LLC | Seattle, WA</p>`

it's magic!